types/index.d.ts

```diff
@@ -12,7 +12,8 @@
  * If you are seeing this from the future, please, send us your cutting-edge technology :) (if it exists)
  */
 import { Component, ReactNode } from 'react';
-import { NativeMethods, ViewProperties, findNodeHandle } from 'react-native';
+import { NativeMethods, findNodeHandle } from 'react-native';
+import { ViewProperties } from 'deprecated-react-native-prop-types';
```

Comment:
are you sure `ViewProperties` exist in `deprecated-react-native-prop-types`

Reply:
Sorry that was a mistake